coralproject/rte
# [CORL-1211] Link content to match href

## What does this PR do
- Add support for  `linkContentMatchHref` (defaults to `true`). This causes link href to always sync with its content.
- Allow changing the regexp to find links using `linkRegExp`.

## How to test this PR
- Enter a link until it is detected, change content and notice the `href` changes as well
